Runtime/Behaviours/CinemachineStateDrivenCamera.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace Cinemachine
 {
+#if CINEMACHINE_UNITY_ANIMATION
```

Comment:
Need at the top:
#if !UNITY_2019_3_OR_NEWER
#define CINEMACHINE_UNITY_ANIMATION
#endif